code/modules/vending/citadelplush.dm

```diff
@@ -15,8 +15,8 @@
 				   /obj/item/toy/plush/moth = 5)
 	contraband = list(/obj/item/toy/plush/nukeplushie = 5,
 				   /obj/item/toy/plush/narplush = 5,
-				   /obj/item/toy/plush/plushvar = 5)
-	premium = list(/obj/item/toy/plush/qwerty = 5,)
+				   /obj/item/toy/plush/plushvar = 5,)
```

Comment:
At the end of a list, you don't need to put a comma, everything else is fine

Reply:
Done